### DISCUSSION TITLE
I can use ALL file?

### DISCUSSION THREAD

**Discussion by ELIA1233** [kimlimjustin/xplorer#109] [Q&A] [answered]
I can use ALL file with this explorer?

**Comment by kimlimjustin** [comment, marked as answer]
Almost all, except for phone files, you can't open it automatically (You have to mount it and do some stuff again) (#6)

**Reply by ELIA1233** [reply]
Ok if you can open all files (incluse the phone file) its a VERY GOOD explorer Good luck!
